### PR TITLE
feat: event hub

### PR DESCRIPTION
**Pull request by owlleg6**

### CODE REVIEW

File: output.tf [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  }
+
+output "name" {
+  value       = azurerm_eventhub_namespace.this.name
+  description = "Azurerm eventhub namespace name"
+}
+
+output "access-primary-key" {
+  value       = azurerm_eventhub_namespace_authorization_rule.this.primary_key
+  description = "Eventhub primary key"
```
Comment (owlleg6) [inline output.tf]: Event hub's namespace shared access key

File: output.tf [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+output "access-key-name" {
+  value       = azurerm_eventhub_namespace_authorization_rule.this.name
+  description = "Eventhub shared access key name"
```
Comment (owlleg6) [inline output.tf]: Event hub's namespace shared access key name

File: output.tf [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+output "access-endpoint" {
+  value       = "sb://${azurerm_eventhub_namespace.this.name}.servicebus.windows.net/"
+  description = "Eventhub shared access key name"
```
Comment (owlleg6) [inline output.tf]: Event hub's namespace endpoint URL

File: main.tf [hunk 4]
```diff
@@ -0,0 +1,47 @@
+resource "random_integer" "this" {
+  min = 99
+  max = 999
+}
+locals {
+  random = "eventhub-${random_integer.this.result}-${var.project}-${var.env}-${var.location}"
+}
+
+resource "azurerm_eventhub_namespace" "this" {
+  name                = local.random
```
Comment (owlleg6) [inline main.tf]: remove locals var

File: main.tf [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  location            = var.location
+  resource_group_name = var.resource_group
+  sku                 = var.sku
+  capacity            = var.capacity
+}
+
+resource "azurerm_eventhub_namespace_authorization_rule" "this" {
+  name                = "eventhub-${var.project}-${var.env}-${var.location}"
```
Comment (owlleg6) [inline main.tf]: change to "namespace"

File: outputs.tf [hunk 6]
```diff
@@ -0,0 +1,24 @@
+output "connection_string" {
+  value       = azurerm_eventhub_namespace_authorization_rule.this.primary_connection_string
+  description = "Event Hub namespace connection string"
+}
+
+output "name" {
```
Comment (ivasm-ko) [inline outputs.tf]: consider renaming the output to a namespace

**Review by ivasm-ko — APPROVED** [review]

File: outputs.tf (hunk 6 above)
Comment (owlleg6) [reply outputs.tf]: done

**Pull request merged**